.github/workflows/publish-next.yml

```diff
@@ -151,7 +151,6 @@ jobs:
           retention-days: 5
 
   publish:
-    needs: [e2e, guides]
```

Comment:
skip this since these tests passed

Reply:
I think we omit RN tests on PRs that don't touch certain folders though, are we sure we want to skip here?